components/coding/ProfileGoalsSection.tsx

```diff
@@ -24,6 +24,7 @@ export default function ProfileGoalsSection({ user }: GoalsSectionProps) {
       },
     }
   );
+  const slicedUserGoals = userGoals.slice(0, 5);
```

Comment:
one small change I'm thinking here - how about we filter to only show current goals, but keep the slice so we show 5 current goals at most? 

I think this makes more sense from a user perspective - what do you think?